app/views/repositories/empty.html.erb

```diff
@@ -33,6 +33,7 @@ See doc/COPYRIGHT.rdoc for more details.
              message: l('repositories.errors.empty_repository')) %></p>
   </div>
 </div>
+<%= render partial: 'checkout_instructions', locals: { repository: @repository, instructions: @instructions } %>
```

Comment:
This is nitpickery, but I'd expect the instructions above the empty message. Probably because I am used to empty tables being displayed as

> some text above
> *no data to display*